src/io/pulseaudio_io.ml

```diff
@@ -71,12 +70,14 @@ class output ~infallible ~start ~on_start ~on_stop ~kind p =
 
     val mutable stream = None
 
+    method channels = AFrame.channels_of_kind self#kind
```

Comment:
Why not just inlining this below? Seems like the only place it's used at and adding extra methods to objects makes typing annoying, even when `private`..

Reply:
The number of channels might be used multiple times (it is not the case here, but I am using the same coding pattern everywhere) and I find it clearer this way. I'll add `private` though. Thanks much for the comment though, I have spotted many places where we were using the kind too early!...

Reply:
Actually not, it seems that I can remove delaying channels computations from the diff! :)